### PR TITLE
4 group event models closes #4

### LINKED ISSUE #4: Group, Event Models
Create Models for:

Group
Event

Refer to ERD

### PR DESCRIPTION
**Pull request by CB6670**
Created the Event and Group Models as well as added them to the db connection.

### CODE REVIEW

File: server/db/models/event.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        zipcode: {
+            type: INTEGER,
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+    }
+);
```
Comment (nicknapoli82) [inline server/db/models/event.js:68]: @justin-cook-developer got me on this one as well. Pretty sure you still need
```        sequelize: db,
        modelName: 'user',```
defined for this model

**Review by nicknapoli82 — COMMENTED** [review]
Should we have an event duration added as well? I just thought about that..

File: server/db/models/event.js (hunk 1 above)
Comment (CB6670) [reply server/db/models/event.js:68]: Oh yeah, I'll get on it.

File: server/db/models/event.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        name: {
+            type: STRING,
+            allowNull: false,
+            validate: {
+            notEmpty: true
+            }
+        },
+        description: {
+            type: TEXT,
+            allowNull: true
```
Comment (justin-cook-developer) [inline server/db/models/event.js]: I think we should require descriptions for events.

File: server/db/models/group.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+        description: {
+            type: TEXT,
+            allowNull: true
+        },
+        sequelize: db,
```
Comment (justin-cook-developer) [inline server/db/models/group.js]: The sequelize and model name keys need to be in their own object. See my user model.

**Review by justin-cook-developer — DISMISSED** [review]

File: server/db/models/event.js (hunk 2 above)
Comment (CB6670) [reply server/db/models/event.js]: Sounds good, I'll change it.

File: server/db/models/group.js (hunk 3 above)
Comment (CB6670) [reply server/db/models/group.js]: Ok, what do you think about Nick's suggestion as well? Do we want to have a duration for the event on the database?

Comment (justin-cook-developer) [reply server/db/models/group.js]: I think that sounds good.

**Review by nicknapoli82 — APPROVED** [review]
Looks good to me man. Keep it up :)

**Pull request merged**